docs/example-entity-definition.yml

```diff
@@ -44,3 +44,10 @@ compositeMetrics:
 goldenTags:
   - tagNameA
   - tagNameB
+
+# Defines whether the entities of this type are automatically deleted after a certain expirationTime without receiving telemetry
+expiration:
```

Comment:
thinking out loud. Should this be schema? or configuration? or something vague enough that will allow us to provide more features in the future?

Reply:
We could add this under a `configuration` section as you say, or keep adding top level configs that might be unrelated between them (for example, maybe some tag configs in the future?).

Reply:
whatever you prefer. I'm trying to think on ways to do as less as breaking changes as possible. But we can live with this I think

Reply:
yeah 😅 maybe better safe than sorry, will do